test/functional/smart_answers_controller_test.rb

```diff
@@ -196,65 +200,83 @@ def teardown
     end
 
     context "A/B testing" do
-      setup do
-        content_item = {
-          "links" => {
-            "taxons" => [
-              {
-                "title" => "A Taxon",
-                "base_path" => "/a-taxon",
-              }
-            ],
-          },
-        }
-
-        Services.content_store.expects(:content_item)
-          .with("/smart-answers-controller-sample")
-          .returns(content_item)
-
-        navigation_helper = GovukNavigationHelpers::NavigationHelper.new(content_item)
-        navigation_helper.stubs(:breadcrumbs).returns(breadcrumbs: ['NormalBreadcrumb'])
-        navigation_helper.stubs(:taxon_breadcrumbs).returns(breadcrumbs: ['TaxonBreadcrumb'])
-        GovukNavigationHelpers::NavigationHelper.expects(:new)
-          .with(content_item)
-          .returns(navigation_helper)
-      end
-
-      should "show normal breadcrumbs by default" do
-        get :show, id: 'smart-answers-controller-sample'
-
-        assert_match(/NormalBreadcrumb/, response.body)
-        refute_match(/TaxonBreadcrumb/, response.body)
-        sidebar = Nokogiri::HTML.parse(response.body).at_css(".related-container")
-        refute_match(/A Taxon/, sidebar)
-      end
+      context "pages under A/B test" do
```

Comment:
NB: I recently swapped this pattern out for a neater one in [`frontend`](https://github.com/alphagov/frontend/commit/f03ea746049f65d144844b065865022a6977ae6e).

Obviously fine to leave unless you feel like being a particularly good citizen.